test/javascripts/turbolinks_test.coffee

```diff
@@ -163,6 +174,12 @@ describe 'Turbolinks', ->
           assert(Turbolinks.fullPageNavigate.called, 'Should perform a full page refresh.')
           done()
 
+      it 'changed assets do not update the browser history stack', (done) ->
```

Comment:
Maybe `does not update the browser history stack in cases where it will refresh`